frontend/src/container/AdvancedSearch.jsx

```diff
@@ -333,21 +282,9 @@ export class AdvancedSearch extends React.Component {
 
   loadSearchUrl = () => {
     const searchObject = parseSearchUrl(qs.parse(this.props.location.search))
-    let currentSearchLength = 0
-
-    if (searchObject.searchType === ADVANCED_SEARCH_TYPE.CONTENT) {
-      currentSearchLength = this.props.contentSearch.resultList.length
-    }
-    /*
-      if (searchObject.searchType === ADVANCED_SEARCH_TYPE.USER) {
-        currentSearchLength = props.userSearch.resultList.length
-      }
-
-      if (searchObject.searchType === ADVANCED_SEARCH_TYPE.SPACE) {
-        currentSearchLength = props.spaceSearch.resultList.length
-      }
-    */
-    this.getSearchResult(searchObject, currentSearchLength)
+    this.setState({ searchType: searchObject.searchType })
```

Comment:
Maybe check here if searchType is in Object.values(ADVANCED_SEARCH_TYPE). If not, you could fallback to contents, and maybe push a new "correct" location.

Reply:
fix